src/schema/objects/metadata.yaml

```diff
@@ -3563,6 +3799,23 @@ VascularCrushingVENC:
       items:
         type: number
         unit: cm/s
+VolumeAffineMatrix:
+  name: VolumeAffineMatrix
```

Comment:
Could be this somehow related with the [ChunkTransformationMatrix](https://bids-specification.readthedocs.io/en/stable/glossary.html#chunktransformationmatrix-metadata)?

Reply:
I don't believe this equivalent to what VolumeAffineMatrix is. @wtclarke could you weigh in here?

Reply:
Fundamentally I think that the two encode the same thing. However, `Chunk` is not a term that anyone uses in MRS, so it is going to confuse users as to its purpose. Instead this gives an explicit description of what it's used for and relates it to a real-world user set parameter `VOI`.